content/en/releases/platforms.md

```diff
@@ -0,0 +1,310 @@
+periodic tests, watch them closely and raise the flag when things break. They
+will also have to help debug and fix any platform specific issues as well.
+
+Creating custom [testgrid][4] dashboards can help to monitor platform specific
+tests.
+
+[4]: https://testgrid.k8s.io
+
+#### Step 3: Releasing
+
+The first 2 steps provide a reasonable expectation that there are people taking
+care of a supported platform and it works in a reproducible environment.
+
+Getting to the next level is a big jump from here. We are talking about real
+users who are betting their business literally on the work we are doing here. So
+we need guarantees around "can we really ship this!?" question.
+
+Specifically we are talking about a set of CI jobs in our release-informing and
+release-blocking tabs of our testgrid. The Kubernetes release team has a "CI
+signal" team that relies on the status(es) of these jobs to either ship or hold
+a release. Essentially, if things are mostly red with occasional green, it would
+be prudent to not even bother making this architecture as part of the release.
+CI jobs get added to release-informing first and when these get to a point where
+they work really well, then they get promoted to release-blocking.
+
+The main problem here is once the project starts to ship something, users will
+start to rely on it. While it's straight forward to setup a CI job as a one time
+thing, it's a totally one to consistently maintain them over time. What SIG
+Release is looking for is a strong green CI signal for release managers to cut a
+release and for folks to be able to report problems and them getting addressed.
+This also includes [conformance testing][2] to ensure that the supported
+platform behaves as intended. This can be done by working with SIG Architecture
+as part of the [conformance sub project][3] in addition to testing and release.
+
+[2]: https://github.com/cncf/k8s-conformance
+[3]: https://github.com/kubernetes/community/tree/master/sig-architecture#conformance-definition
+
+#### Step 4: Finishing
+
+If you got this far, you really have made it! You have a clear engagement with
+the community, you are working seamlessly with all the relevant SIGs, you have
+your content in the Kubernetes release and get end users to adopt your
+architecture. Having achieved conformance, you will gain conditional use of the
+Kubernetes trademark relative to your offerings.
+
+### Deprecating and removing supported platforms
+
+Supported platforms may be considered as deprecated for various reasons, for
+example if they are being replaced by new ones, are not actively used or
+maintained any more. Deprecating an already supported platform has to follow a
+couple of steps:
+
+1. The platform deprecation has been announced on the [Kubernetes developer
+   mailing list](https://groups.google.com/a/kubernetes.io/group/dev)
+   and links to an Kubernetes GitHub issue for further discussions and consensus.
```

Comment:
Are we implying that end users should join this mailing lsit?

Reply:
A this should be the announce mailing list.